pysolarmanv5/pysolarmanv5_async.py

```diff
@@ -252,6 +254,23 @@ async def _send_receive_modbus_frame(self, mb_request_frame):
         mb_response_frame = self._v5_frame_decoder(v5_response_frame)
         return mb_response_frame
 
+    def _handle_bogus(self, frame: bytes) -> bytes:
```

Comment:
async already inherits this _handle bogus method from sync version?

Not everyone will be familiar with the discussion in #62, so I wonder should we rename this method to `handle_double_crc_frames()` or similar.